### DISCUSSION TITLE
Welcome to APSIM Next Generation Discussions!

### DISCUSSION THREAD

**Discussion by hol353** [APSIMInitiative/ApsimX#8443] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas / simulations
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lie112** [comment]
Hi everyone, I'm Adam Liedloff and I'm an ecological/agricultural systems modeller with CSIRO based in Darwin, Australia. I am the main developer behind the CLEM (Crop Livestock Enterprise Model) in APSIM Next Generation. I have really enjoyed my journey over the last six years rebuilding an old excel/vb model in APSIM Next Generation with users and APSIM developers. I thought I'd start a few conversations in this discussion area about new functionality I think APSIM would benefit from and have a place for APSIM modellers to discuss the latest .Net and C# features we should take advantage of now that APSIM NG is using the latest stable supported release of .Net.
How does this discussions work? Are we supposed "Create Issue with Discussion" which links the issue to #8443 as prompted to the right so that each discussion has it's own comments or just responded to the initial post as I have done with an introduction seems messy?

**Reply by hol353** [reply]
Thanks Adam. Not sure how this works or whether to reply to your comment like I am now or 'Add a comment'.  I think we'll only create a GitHub Issue when needed i.e. when there is something that needs to change in APSIM.

**Reply by prince-NG** [reply]
Please, I am very new here. I am wondering about Model 5 in APSIM; it is being looked into because I found it very difficult to change the wheat to chickpea. I don't know if the error is from me or the system itself

**Comment by hol353** [comment]
Actually looking at this further, I see you have comment on the 'welcome' discussion. Instead, perhaps you should have created a new discussion at the top level.

**Reply by lie112** [reply]
Got it.. I was taken straight to the Welcome page and didn't realise there was a Discussion thread above it.

**Comment by yashvirchauhan** [comment]
How do I report Julian's day in APSIM Nextgen?

**Reply by ric394** [reply]
`[Clock].Today.DayOfYear` should give you the Julian day.

**Reply by yashvirchauhan** [reply]
Thank you, it worked perfectly. I also decided to use ChatGPT to see if it could help, and it told me to do this. It's also great support for occasional assistance.
